ci/provisioning/post_provision_config_common.sh

```diff
@@ -39,7 +39,7 @@ case "$id" in
             EXCLUDE_UPGRADE+=,fuse
         else
             DISTRO_NAME=el${release%%.*}
-            EXCLUDE_UPGRADE+=,dpdk
+            EXCLUDE_UPGRADE+=,dpdk*
```

Comment:
(lint) Brace expansions and globs are literal in assignments. Quote it or use an array. [SC2125]